### PR TITLE
docs: add quick start guide

### PR DESCRIPTION
**Pull request by tzemanovic**
rendered page: https://github.com/anoma/anoma/blob/tomas/quick-start/docs/src/quick-start.md

### CODE REVIEW

**Review by gabriella-fw — COMMENTED** [review]
I would say "applied in **a** future block" (line 150) and "check **a **future epoch" (line 246) but if everything else is factually correct, then I have no other comments :)

**Review by gabriella-fw — APPROVED** [review]

File: docs/src/quick-start.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ $ tmux
 
 # inside the tmux/or not
 
-$ ./anoman ledger
+$ ./anoma ledger
```
Comment (juped) [inline docs/src/quick-start.md:69]: i think this is just wrong?

**Review by juped — COMMENTED** [review]
should base on tag v0.3.0, not somewhere random

**Comment by tzemanovic** [comment]
I've rebased on v0.3.0

**Pull request merged**